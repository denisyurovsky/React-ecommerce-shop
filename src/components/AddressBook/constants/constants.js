export const LISTBOX_PADDING = 8;
export const DOUBLE_LISTBOX_PADDING = LISTBOX_PADDING * 2;
export const LIMIT_NUMBER_ITEMS = 8;
export const ITEM_SIZE_LG = 48;
export const ITEM_SIZE_MD = 36;
