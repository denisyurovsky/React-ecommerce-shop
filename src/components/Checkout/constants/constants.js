export const PANEL = {
  DELIVERY_ADDRESS: 'Delivery Address',
  PERSONAL_INFORMATION: 'Personal Information',
  PERSONAL_ADDRESS: 'Personal Address',
  PAYMENT_METHOD: 'Payment method',
  ORDER_CONFIRMATION: 'Order confirmation',
};
