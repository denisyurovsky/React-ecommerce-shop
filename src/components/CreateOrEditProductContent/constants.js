export const PRICE_ERROR = 'Only numbers';
export const DESCRIPTION_ERROR = 'Only latin characters';
export const LENGTH_ERROR = 'Should contain at least one symbol';
export const DISCOUNT_ERROR = 'Discount should be less than price';
export const SELECTION_ERROR = 'Choose category';
