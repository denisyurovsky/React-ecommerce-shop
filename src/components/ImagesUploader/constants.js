export const LOSS_OF_IMAGES = 'We lost images';

export const STATUS = {
  CROPPED: 'cropped',
  CROPPING: 'cropping',
  IDLE: 'idle',
};

export const TEST_IMG_URL = 'http://someURL';
