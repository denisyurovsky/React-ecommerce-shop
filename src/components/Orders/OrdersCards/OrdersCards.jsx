import { Typography } from '@mui/material';
import PropTypes from 'prop-types';
import React from 'react';

import { OrderCard } from '../OrderCard/OrderCard';

import styles from './OrdersCards.module.scss';

export const OrdersCards = ({ displayedOrders }) => {
  return displayedOrders.length === 0 ? (
    <Typography align="center" className={styles.noOrders}>
      There are no orders to display, you can order something at any time
    </Typography>
  ) : (
    displayedOrders.map((order) => (
      <OrderCard key={order.id} orderId={order.id} />
    ))
  );
};

OrdersCards.propTypes = {
  displayedOrders: PropTypes.array,
};
