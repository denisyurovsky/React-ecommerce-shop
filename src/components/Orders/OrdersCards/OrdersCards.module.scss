@import "../../../assets/styles/colors.scss";

.noOrders {
  color: $grey700;
  font-size: 24px !important;
}
