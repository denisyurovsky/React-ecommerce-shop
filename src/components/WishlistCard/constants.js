export const DEFAULT_LABEL_INPUT = 'Wishlist Name';
export const ERROR_LABEL_INPUT = 'Wishlist name should be unique';
