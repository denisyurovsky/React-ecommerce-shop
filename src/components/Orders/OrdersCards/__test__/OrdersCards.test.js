import { rest } from 'msw';
import { setupServer } from 'msw/node';
import React from 'react';

import { orderState } from '../../../../helpers/constants/orderStatus';
import { ordersDto } from '../../../../test-utils/dto/ordersDto';
import testProducts from '../../../../test-utils/dto/productsDto';
import renderWith from '../../../../test-utils/renderWith';
import { OrdersCards } from '../OrdersCards';

const initialUser = {
  user: {
    id: 1,
    orders: ordersDto,
  },
};

let serverUser = {
  id: 1,
  orders: ordersDto,
  isLoading: false,
  errorOccurred: false,
};

const handlersFulfilled = [
  rest.post('/confirm-order', (req, res, ctx) => {
    const { orderId } = req.body;

    const now = new Date();

    return res(
      ctx.json({
        orderStatus: orderState.DELIVERED,
        orderId,
        deliveredAt: now,
      }),
      ctx.status(200)
    );
  }),
  rest.post('/cancel-order', (req, res, ctx) => {
    const { orderId } = req.body;

    return res(
      ctx.json({ orderId, orderStatus: orderState.CANCELLED }),
      ctx.status(200)
    );
  }),
  rest.get('/orders', (req, res, ctx) => {
    const { userId } = req.body;

    const response = [];

    ordersDto.forEach((order) => {
      if (order.userId === userId) {
        response.push(order);
      }
    });

    return res(ctx.json(response), ctx.status(200));
  }),
  rest.get('/users/1', (req, res, ctx) => {
    return res(ctx.json(serverUser), ctx.status(200));
  }),
  rest.get('/users/:userId', (req, res, ctx) => {
    return res(ctx.json(serverUser), ctx.status(200));
  }),
  rest.get('/products/', (req, res, ctx) => {
    const response = [];

    if (req.url.searchParams) {
      req.url.searchParams.forEach((item) => {
        response.push(testProducts[item]);
      });
    }

    return res(ctx.json(response), ctx.status(200));
  }),
];

const serverFulfilled = setupServer(...handlersFulfilled);

describe('OrdersCards component', () => {
  describe('snapshots with orders and without orders', () => {
    beforeEach(() => {
      serverFulfilled.listen();
    });
    afterEach(() => {
      serverFulfilled.close();
    });

    it('Renders a valid orderCard snapshot without orders', () => {
      const { asFragment } = renderWith(<OrdersCards displayedOrders={[]} />, {
        orders: {
          orders: ordersDto,
        },
        user: initialUser,
      });

      expect(asFragment()).toMatchSnapshot();
    });

    it('Renders a valid orderCard snapshot with orders', () => {
      const { asFragment } = renderWith(
        <OrdersCards displayedOrders={ordersDto} />,
        {
          orders: {
            orders: ordersDto,
          },
          user: initialUser,
        }
      );

      expect(asFragment()).toMatchSnapshot();
    });
  });
});
