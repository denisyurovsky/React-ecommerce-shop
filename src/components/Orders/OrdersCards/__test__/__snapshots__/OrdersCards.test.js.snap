// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersCards component snapshots with orders and without orders Renders a valid orderCard snapshot with orders 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #2
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          10 000 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 January 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #3
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark2 css-ahj2mt-MuiTypography-root"
          >
            Waiting for delivery
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          714 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 March 2022
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #4
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark3 css-ahj2mt-MuiTypography-root"
          >
            Delivered
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          539 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 September 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #5
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark4 css-ahj2mt-MuiTypography-root"
          >
            Cancelled
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          274,50 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #6
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          310 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #7
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          310 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #8
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          111 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #9
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark3 css-ahj2mt-MuiTypography-root"
          >
            Delivered
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          310 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #10
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark4 css-ahj2mt-MuiTypography-root"
          >
            Cancelled
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          111 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #11
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          310 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #12
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          111 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 April 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #13
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          310 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 March 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #14
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          111 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 August 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #15
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          310 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 January 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #16
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          740 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardActions-root MuiCardActions-spacing collapseButton css-1t6e9jv-MuiCardActions-root"
    >
      <div
        class="outerInformation MuiBox-root css-0"
      >
        <div
          class="dateInformation MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 orderDate css-ahj2mt-MuiTypography-root"
          >
            Ordered on  19 February 2021
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
          >
            Order #17
          </p>
        </div>
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 deliveryMark deliveryMark1 css-ahj2mt-MuiTypography-root"
          >
            Waiting for payment
          </p>
        </div>
      </div>
      <div
        class="priceInformation MuiBox-root css-0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 orderPrice css-ahj2mt-MuiTypography-root"
        >
          3 010 $  
        </p>
        <svg
          aria-expanded="false"
          aria-hidden="true"
          aria-label="show more"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium showMoreButton css-i4bv87-MuiSvgIcon-root"
          data-testid="ExpandMoreIcon"
          expand="false"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
          />
        </svg>
      </div>
    </div>
  </div>
  <div
    class="Toastify"
  />
</DocumentFragment>
`;

exports[`OrdersCards component snapshots with orders and without orders Renders a valid orderCard snapshot without orders 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter noOrders css-1uwgr7b-MuiTypography-root"
  >
    There are no orders to display, you can order something at any time
  </p>
  <div
    class="Toastify"
  />
</DocumentFragment>
`;
